repository/src/main/java/org/apache/atlas/repository/store/graph/v2/tasks/MeaningsTask.java

```diff
@@ -60,6 +61,7 @@ public AtlasTask.Status perform() throws Exception {
             RequestContext.get().setUser(userName, null);
             try {
                 setStatus(IN_PROGRESS);
+                setAssetsCountToPropagate(1234L);
```

Comment:
Let's remove this code which was used for debugging.

Reply:
Removed, thanks for pointing out